### DISCUSSION TITLE
I am new here, how do you like copilot?

### DISCUSSION THREAD

**Discussion by tomivd** [community/community#45601] [New to GitHub] [answered]
I have signed up for a free account. And am highly interested in copilot. I like programming and would love someone’s take on how well they like copilot. And if it’s worth the costs. Albeit the costs is not high. Is it a good value is what I’m concerned with.  Thank you in advance.

**Comment by jge162** [comment, marked as answer]
GitHub copilot is a great option for anyone that does programming. I use it personally and think it is of great value personally. I am a student so there is no cost to me. But, I 100% would pay the monthly fee to have it. 
Here is link to [Copilot documentation](https://docs.github.com/en/billing/managing-billing-for-github-copilot/about-billing-for-github-copilot)
<img width="792" alt="Screenshot 2023-01-28 at 12 00 26 AM" src="https://user-images.githubusercontent.com/31228460/215254515-09587781-1983-4647-8f2d-c430a0d4efed.png">

**Reply by jge162** [reply]
Anything else please do not hesitate to ask and welcome

**Reply by tomivd** [reply]
Tysm. Makes the decision very easy for me.

**Comment by dallen91801** [comment]
No $$$$$ figures in this thread WTH?
